test/.eslintrc.yaml

```
@@ -68,3 +69,24 @@ rules:
 # Global scoped methods and vars
 globals:
   WebAssembly: false
+
+overrides:
+  - files:
+      - addons/*/*.js
+      - async-hooks/*.js
+      - common/*.js
+      - es-module/*.js
+      - es-module/*.mjs
+      - internet/*.js
+      - js-native-api/*/*.js
+      - known_issues/*.js
+      - message/*.js
+      - node-api/*/*.js
+      - parallel/*.js
+      - parallel/*.mjs
+      - pseudo-tty/*.js
+      - pummel/*.js
+      - sequential/*.js
+      - sequential/*.mjs
+    rules:
+      comma-dangle: off
```

Would it be better to use the current value, which we'd copy from the main `.eslintrc.js` file?

```suggestion
      comma-dangle: [error, {
        arrays: always-multiline,
      exports: only-multiline,
      functions: only-multiline,
      imports: only-multiline,
      objects: only-multiline,
    }],
```